builder/parser/testfiles/continueIndent/Dockerfile

```diff
@@ -13,7 +13,7 @@ world
 RUN echo hello \
 goodbye\
 frog
-RUN echo hello  \  
```

Comment:
I think this one was here on purpose to make sure the builder handles trailing spaces consistently.